conf/up4.bess

```diff
@@ -438,19 +440,46 @@ check_spgwu_ip = " and dst host " + \
     " or ".join(str(x) for x in access_ip)
 check_gtpu_port = " and udp dst port 2152"
 check_gtpu_msg_echo = " and udp[9] = 0x1"
+check_gtpu_msg_echo_res = " and udp[9] = 0x2"
 
 # Echo filter
 uplink_echo_filter = {"priority": GTPUEchoGate, "filter": check_ip +
                       check_spgwu_ip + check_gtpu_port +
                       check_gtpu_msg_echo, "gate": GTPUEchoGate}
 accessFastBPF.add(filters=[uplink_echo_filter])
 
+# Echo Response filter
+uplink_echo_res_filter = {"priority": GTPUEchoResGate, "filter": check_ip +
+                          check_spgwu_ip + check_gtpu_port +
```

Comment:
We should modify spgwu to upf in all places.

Reply:
Yes, I already have those changes in my local machine for quite some time, and I was planning to open a PR just for that after other PRs such as `Enable GTPu Path Monitoring` are merged to avoid conflicts with other PRs